AdvancedCalculator/Program.cs

```diff
@@ -1,4 +1,6 @@
 ﻿using AdvancedCalculator.Infrastucture.Calculators;
 
+int arry = Convert.ToInt32(Console.ReadLine());
```

Comment:
arry? co to jest, nazwa powinna przedstawiać to co jest w środku zmiennej

Reply:
zrobione